### DISCUSSION TITLE
Panel: Simple time picker per panel with Hour, Day, Week shift options

### DISCUSSION THREAD

**Discussion by m50434** [grafana/grafana#37430] [Dashboards, Panels & Visualizations]
What would you like to be added:
I would appreciate the possibility to get a timeshift for each panel (not for the whole dashboard) with some buttons (so you can shift the time with one click backwards or forwards in a special range, i.e. day hour or month.

Why is this needed:
It would be nice to compare i.e. the current day with the day before immediately.
Or just to go through your data quite fast.

Here is an example: There are some buttons.

![grafik](https://user-images.githubusercontent.com/24473301/127739448-62a85641-7e06-4500-9ec5-42d4a72dc6ff.png)

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
